src/test/java/bio/terra/service/filedata/DrsServiceTest.java

```diff
@@ -188,6 +213,145 @@ public void testLookupNegative() {
         () -> drsService.lookupObjectByDrsId(authUser, azureDrsObjectId, false));
   }
 
+  @Test
+  public void testLookupSnapshot() {
+    drsService.lookupSnapshotForDRSObject(googleDrsObjectId);
```

Comment:
Can we assert that this retrieves the expected snapshot?

Reply:
Good idea - Added!